doc/glossary.rst

```diff
@@ -0,0 +1,305 @@
+        their associated estimation procedures used to analyze the differences
+        among means.
+
+    AUC
+        `Area under the curve`_.
+
+    BIDS
+    bids
+        `Brain Imaging Data Structure`_ is a simple and easy to adopt way
+        of organizing neuroimaging and behavioral data.
+    
+    BOLD
+    Bold
+    bold
+        Blood oxygenation level dependent. This is the kind of signal measured
+        by functional Magnetic Resonance Imaging.
+
+    CanICA
+        `Canonical independant component analysis`_.
+
+    contrast
+    contrasts
+        A `contrast`_ is a linear combination of variables (parameters or
+        statistics) whose coefficients add up to zero, allowing comparison
+        of different treatments.
+
+    Decoding
+        `Decoding`_ consists in predicting, from brain images, the conditions
+        associated to trial.
+
+    EEG
+        `Electroencephalography`_ is a monitoring method to record electrical
+        activity of the brain.
+    
+    EPI
+        Echo-Planar Imaging. This is the type of sequence used to acquire
+        functional or diffusion MRI data.
+
+    FDR correction
+        `False discovery rate`_ controlling procedures are designed to control
+        the expected proportion of "discoveries" (rejected null hypotheses)
+        that are false (incorrect rejections of the null).
+
+    FIR
+        Finite impulse response. This is a type of free-form temporal filter
+        that is used to link neural activity with hemodynamic response, when
+        there is uncertainty on the true model.
+
+    fMRI
+    fmri
+        Functional magnetic resonance imaging is based on the fact that 
+        when local neural activity increases, increases in metabolism and
+        blood flow lead to fluctuations of the relative concentrations of
+        oxyhaemoglobin (the red cells in the blood that carry oxygen) and
+        deoxyhaemoglobin (the same red cells after they have delivered the
+        oxygen). Oxyhaemoglobin and deoxyhaemoglobin have different magnetic
+        properties (diamagnetic and paramagnetic, respectively), and they
+        affect the local magnetic field in different ways.
+        The signal picked up by the MRI scanner is sensitive to these
+        modifications of the local magnetic field.
+
+    FPR correction
+    FPR
+    fpr
+        False positive rate correction. This refers to the methods employed to
+        correct false positive rates such as the Bonferroni correction which
+        divides the significance level by the number of comparisions made.
+
+    FREM
+        `FREM`_ means "Fast ensembling of REgularized Models". It uses an implicit
+        spatial regularization through fast clustering and aggregates a high 
+        number of estimators trained on various splits of the training set, thus 
+        returning a very robust decoder at a lower computational cost than other 
+        spatially regularized methods. 
```

Comment:
I think that you should cite the paper: https://www.sciencedirect.com/science/article/abs/pii/S1053811917308182